### PR TITLE
add `/statusbar tabmode actlist`

### PR DESCRIPTION
**Pull request by sjaeckel**
The existing way how active tabs are displayed didn't allow showing more than 10 tabs. This patch adds a mode where the statusbar shows a comma-separated list of tabs which were active since the last time viewed.
This view is inspired by how `irssi` shows the active tabs, therefore it is also called `actlist`.

### CODE REVIEW

**Review by jubalh — CHANGES_REQUESTED** [review]
(just commenting to have a better overview)
`/statusbar tabmode default|actlist` as discussed privately.

**Comment by sjaeckel** [comment]
> (just commenting to have a better overview) `/statusbar tabmode default|actlist` as discussed privately.

I'll wait until #1817 is merged, since that PR touches some of the same parts of the code.

**Review by jubalh — APPROVED** [review]

**Comment by jubalh** [comment]
Thanks!

**Pull request merged**
